templates/tease-taxonomy-action.twig

```diff
@@ -0,0 +1,59 @@
+<li id="result-row-{{ post.ID }}" class="d-flex search-result-list-item">
+	<div class="search-result-item-image search-result-item-image-max-width">
+		{% include "blocks/image.twig" with { post: post, dummy_thumbnail: dummy_thumbnail, data_ga_category: 'Actions List', classname: '' } %}
+	</div>
+
+	<div id="tease-{{ post.ID }}" class="search-result-item-body tease tease-{{ post.post_type }}">
+		<div class="search-result-tags top-page-tags">
```

Comment:
For more info: https://greenpeace-gpi.slack.com/archives/G015K63081W/p1667920973170169